scripts/toaster_configuration

```diff
@@ -189,6 +189,12 @@ EOF
 }
 
 create_toaster_venv () {
+    if [ -f /etc/redhat-release ]; then
+        # If its RHEL 7 there is a bug in virtualenv which has to be tweaked
+        if [ $(cat /etc/redhat-release  | awk {'print $7'} | cut -d "." -f1) -eq 7 ]; then
```

Comment:
Worth noting that redhat-release does exist in all redhat variants, not only rhel. I don't think it'll be an issue, since in e.g. CentOS the version number is field 3 rather than 7, but just to be aware of it.